### PR TITLE
[PRDP-36][PRDP-45] feat: Developed Azure Functions and configured for deploy

### PR DESCRIPTION
**Pull request by svariant**
#### List of Changes

Development:
- Defined receipt-to-io function
- Added health check Info function
- Defined unit tests for receipt-to-io function
- Generated IO client with OpenAPI spec
- Defined notifier-retry function
- Defined unit tests for notifier-retry function
- Implemented Azure Storage Queue client
- Commented code
- Written README

Deploy:
- Configured helm details
- Removed unused devops pipelines

#### Motivation and Context

Development:
[Issue PRDP-36](https://pagopa.atlassian.net/browse/PRDP-36?atlOrigin=eyJpIjoiMWYyZGNiNzllMTI4NDIxODhjMjQ2ZTg3NjgyMDU4OGEiLCJwIjoiaiJ9)
Dev Deploy:
[Issue PRDP-45](https://pagopa.atlassian.net/browse/PRDP-45?atlOrigin=eyJpIjoiNmIxZTk1NGY4ZjUyNDZmYmFmYzFhOGY1NzIyYzFkOTEiLCJwIjoiaiJ9)

#### How Has This Been Tested?

Unit tests have been defined for functions and clients

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: helm/values-prod.yaml [hunk 1]
```diff
@@ -61,13 +61,48 @@ microservice-chart:
           type: Utilization # Allowed types are 'Utilization' or 'AverageValue'
           value: "75"
   envConfig:
-    WEBSITE_SITE_NAME: "pagopa<project-name>" # required to show cloud role name in application insights # TODO
+    WEBSITE_SITE_NAME: "pagopareceiptpdfnotifier" # required to show cloud role name in application insights
     FUNCTIONS_WORKER_RUNTIME: "java"
+    NOTIFIER_QUEUE_TOPIC: "pagopa-d-weu-receipts-queue-receipt-io-notifier-error"
+    COSMOS_RECEIPT_SERVICE_ENDPOINT: "https://pagopa-d-weu-receipts-ds-cosmos-account.documents.azure.com:443/"
```
Comment (pasqualespica) [inline helm/values-prod.yaml:67]: attention to use DEV var secret into PROD env

**Review by pasqualespica — CHANGES_REQUESTED** [review]

**Comment by pasqualespica** [comment]
@svariant @alessio-cialini according to our comments
_This PR is the final step of development (before improvements and bug fixing), therefore it contains others PRs such as
https://github.com/pagopa/pagopa-receipt-pdf-notifier/pull/2 and https://github.com/pagopa/pagopa-receipt-pdf-notifier/pull/3_
i thins coold be usefull merge the following one PR in one of the previous , es [this one](https://github.com/pagopa/pagopa-receipt-pdf-notifier/pull/2)
and close it

File: helm/values-prod.yaml (hunk 1 above)
Comment (svariant) [reply helm/values-prod.yaml]: Changed to correct values

**Comment by svariant** [comment]
Cloed and merged PRs https://github.com/pagopa/pagopa-receipt-pdf-notifier/pull/3 and https://github.com/pagopa/pagopa-receipt-pdf-notifier/pull/2 into this PR

**Comment by svariant** [comment]
As reminder we can expose all info service on apim receipt status product, to our [internal status pages](https://pagopa.github.io/pagopa-status-page/#/)

_Originally posted by @pasqualespica in https://github.com/pagopa/pagopa-receipt-pdf-notifier/pull/2#discussion_r1255752788_

**Review by pasqualespica — APPROVED** [review]
great work 👍

**Pull request merged**